packages/replay/metadata/test.ts

```diff
@@ -10,27 +9,49 @@ const {
   optional,
   string,
   define,
-} = require("superstruct");
+  Struct,
+} from "superstruct";
 const isUuid = require("is-uuid");
 
 import { UnstructuredMetadata } from "./types";
 
 const VERSION = 1;
 
-const versions: Record<number, Struct> = {
+const versions: Record<number, Struct<any, any>> = {
   1: object({
     file: optional(envString("RECORD_REPLAY_METADATA_TEST_FILE")),
     path: optional(array(string())),
     result: defaulted(
       enums(["passed", "failed", "timedOut"]),
       firstEnvValueOf("RECORD_REPLAY_METADATA_TEST_RESULT")
     ),
+    steps: optional(
```

Comment:
Hesitant to prescribe the format of this yet because we don't really know what we need. I think this is pretty safe minimum set though and maps cleanly from playwright

Reply:
Skipping "steps" for now but we can bring it back later when we know how to populate it from cypress